recipe-server/normandy/health/api/views.py

```diff
@@ -15,42 +17,51 @@
 
 
 logger = logging.getLogger(__name__)
-_commit = None
-_tag = None
 
 
+@lru_cache(maxsize=1)
 def get_version_info():
```

Comment:
This could also use a docstring explaining that it pulls data about the current build. It'd be great to reference the filename of the script generating the files, too.